test/oss_test.cpp

```diff
@@ -20,120 +20,172 @@
 #include <unistd.h>	
 #include <gtest/gtest.h>
 
+using namespace emeralddb::oss;
+using namespace std;
+using namespace testing;
+
+
+class ossSocketTest : public Test {
+public:
+  ossSocketTest() {}
+  ~ossSocketTest() {}
+
+protected: 
+  virtual void SetUp() {
+    cout << "before test" << endl; 
+    _port = genRandomPort(); 
+    _timeout = 6; 
+    ossPtr = getInstance(); 
+  }
+
+  virtual void TearDown() {
+    cout << "after test" << endl; 
+    release(ossPtr);
+    ossPtr = nullptr; 
+    ASSERT_EQ(nullptr, ossPtr); 
```

Comment:
gtest supports `ASSERT_EQ` `ASSERT_TRUE` `ASSERT_FALSE` `EXPECT_EQ` not support `ASSERT_NQ` 
also `ASSERT_LT` `ASSERT_GT`